packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -1519,7 +1519,11 @@ abstract class FlutterCommand extends Command<void> {
   ApplicationPackageFactory? applicationPackages;
 
   /// Gets the parsed command-line option named [name] as a `bool`.
-  bool boolArg(String name) => argResults?[name] as bool? ?? false;
+  @Deprecated('Deprecated in favor of boolArg')
```

Comment:
@christopherfujino do we need to mark it deprecated as per the [policy](https://github.com/flutter/flutter/wiki/Tree-hygiene#deprecations)?

Reply:
I asked @Jasguerrero to review this, as it was his proposal.

Reply:
> @christopherfujino do we need to mark it deprecated as per the [policy](https://github.com/flutter/flutter/wiki/Tree-hygiene#deprecations)?

@amanv8060 Unlike the framework the tool is not imported by customers so no need to follow the wiki's policy in this case.

Reply:
Sure, I will remove it.

Reply:
@Jasguerrero, will this be a test-exempt?